docs/application/dotnet/guides/nui/text.md

```diff
@@ -89,156 +97,89 @@ If no font is specified, default styles are used, and a suitable font for displa
 
 Setting a font size programmatically is not ideal for applications that support multiple screen resolutions, and for platforms that support multiple logical font sizes. In addition, making systemwide changes to your font settings override the font sizes that have been programmatically set.
 
-A more flexible approach is to prepare various JSON stylesheets and request a different style for each platform. The [Tizen.NUI.NUIApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIApplication.html) class has constructors that take a stylesheet argument:
+A more flexible approach is to prepare various JSON stylesheets and request a different style for each platform. To change the font style for standard text controls, create `theme.json` file in resources directory and use the following JSON syntax:

```

Comment:
Please add article "the":

A more flexible approach is to prepare various JSON stylesheets and request a different style for each platform. To change the font style for standard text controls, create the `theme.json` file in the resources directory and use the following JSON syntax: